app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, which means it is shared across all instances of `Animal`. This might lead to unexpected behavior if instances are removed from the list in one part of the code and accessed in another. Consider whether this is the intended design.